Cookbook/Technical-Documents/VerticalFrameworks.md

```diff
@@ -0,0 +1,133 @@
+Vertical Frameworks
+===================
+
+## Overview
+
+After modularisation refactoring in [IOSP-338](https://babylonpartners.atlassian.net/browse/IOSP-338), our Babylon App's UI screen framework dependencies are now fully decoupled from each other so called **Vertical Frameworks**.
+It ensures that there are **no dependencies between each vertical** so that faster parallel build and easy maintainance become possible.
+
+<img src="Assets/parallel-build.png" width="500"><br>
+(Example of parallel build in Vertical Frameworks)<br>
+
+
+In order to share each vertical's functionalities among other verticals, e.g. `ClinicalRecordsUI` using `MapsUI` feature, rather than one depending on the other sequentially, we now add **`BabylonVerticalInterfaces.framework`** ([PR-9964](https://github.com/babylonhealth/babylon-ios/pull/9964)) as a shim to **share each vertical's interfaces only** to the other verticals.
+
+<img src="Assets/dependency-v4.12.0.png">
```

Comment:
It does not show image. Probably we should use ![](Assets/dependency-v4.12.0.png)

Reply:
It's visible in https://github.com/babylonhealth/ios-playbook/blob/94a94840c3f5a52175ac64923dbac3008c0db6d0/Cookbook/Technical-Documents/VerticalFrameworks.md , so should be no problem.